### ISSUE TITLE
zscore ci level

### ISSUE THREAD

**Issue by graceannobrien** [metrumresearchgroup/pmparams#70]
If user provides `.zscore`, it does not rename `ci_level` which then carries through all subsequent functions, making final table have incorrect CI name
![Screenshot 2024-10-20 at 1 15 20 PM](https://github.com/user-attachments/assets/30b2b034-d8af-4469-9338-a39a5c4554e9)

**Comment by seth127** [comment]
As noted in this comment on #64, we plan to deprecate the `.zscore` argument and calculate it manually instead, for all values of `.ci`. Note that this will also necessitate changing [`param_notes()$ciEq()`](https://github.com/metrumresearchgroup/pmparams/blob/0.2.2/R/param-notes.R#L65) to _not_ use hard-coded values. (While we're there, delete [this confusing logic](https://github.com/metrumresearchgroup/pmparams/blob/0.2.2/R/param-notes.R#L36-L55).)

The current plan is to replace this with a more generic note, likely using the `$z_{\alpha/2}$` LaTeX (from [here](https://tex.stackexchange.com/a/485393)). @KatherineKayMRG or @kylebaron could one of you write the replacement note that you'd like and put it in a comment here? Thanks in advance.

**Comment by kylebaron** [comment]
Some ideas (if this is what the question is); I've hard-coded `ci = 95` into the output here, but we will `glue()` or `paste()` that in based on what is passed for `ci`. @KatherineKayMRG thoughts?

<img width="587" alt="Image" src="https://github.com/user-attachments/assets/5b028484-7766-4efe-9887-f0dfc46af757" />

**Comment by KatherineKayMRG** [comment]
Oooph good question. I think I'd lean towards option 4 and have as little as possible subscripted - it gets small in the report. But I'm open to other peoples preferences too. 

@timwaterhouse you've commented on these notes in other PRs - any strong preferences here

**Comment by timwaterhouse** [comment]
@KatherineKayMRG We're just talking about the table footnote now, right? I prefer option 4 as well. I think that's the cleanest while still providing all the information.

**Comment by KatherineKayMRG** [comment]
Yes sorry, there was a discussion last week about re-factoring how CIs are calculated to avoid hard coding the CI. The package will now calculate the Z-score (defaulting to 95% CI still) so the footer also needs updating to reflect how it will be calculated. 

It will cause some minor difference in CI estimates because the current hard coded z-scores for 95% and 90% CIs were rounded to 3 sd - calculating them will have increased precision and hence cause slight differences in CI estimates. We just want to be clear what is being returned by the package version
